src/vdaf.rs

```diff
@@ -483,12 +483,37 @@ pub mod test_utils {
         let mut verify_key = [0; SEED_SIZE];
         rng.fill(&mut verify_key[..]);
 
-        let mut agg_shares: Vec<Option<V::AggregateShare>> = vec![None; vdaf.num_aggregators()];
-        let mut num_measurements: usize = 0;
+        let mut sharded_measurements = Vec::new();
         for measurement in measurements.into_iter() {
-            num_measurements += 1;
             let nonce = rng.gen();
```

Comment:
```suggestion
            let nonce = random();
```

nit: lets us get rid of the `rng` variable, after removing `verify_key`. I would also suggest we hoist up `use rand::prelude::*` to the mod level, for some dedupe between the two `run_vdaf` functions.